guides/source/debugging_rails_applications.md

```diff
@@ -232,23 +231,31 @@ irb(main):003:0> Article.pamplemousse
 => #<Comment id: 2, author: "1", body: "Well, actually...", article_id: 1, created_at: "2018-10-19 00:56:10", updated_at: "2018-10-19 00:56:10">
 ```
 
-Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call. This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
+Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call e.g. `↳ app/models/article.rb:5`.
+
+This can help you identify and address performance problems caused by N+1 queries: i.e. single database queries that generate multiple additional queries.
 
-Verbose query logs are enabled by default in the development environment logs after Rails 5.2.
+Verbose query logs are [enabled by default](configuring.html#config-active-record-verbose-query-logs) in the development environment logs.
 
 WARNING: We recommend against using this setting in production environments. It relies on Ruby's `Kernel#caller` method which tends to allocate a lot of memory in order to generate stacktraces of method calls. Use query log tags (see below) instead.
 
 ### Verbose Enqueue Logs
 
 Similar to the "Verbose Query Logs" above, allows to print source locations of methods that enqueue background jobs.
 
-It is enabled by default in development. To enable in other environments, add in `application.rb` or any environment initializer:
+Verbose enqueue logs are enabled by default in the development environment logs after Rails 7.1.
```

Comment:
I find `after` a bit unclear here.
Maybe "starting with" (which we often use in the configuration guide?):
```suggestion
Verbose enqueue logs are enabled by default in the development environment logs starting with Rails 7.2.
```

Reply:
> So I think we can remove the reference to the version.

Removing it goes against my specific need. I had a pre-7.1 rails app. I cannot be the only one? I had gone so far as to check against the original PR. In light of the aforementioned: pls confirm whether you want this removed?

Reply:
If we show the version it was released to every feature the guides would become unreadable.

Maybe expand the 7.1 release notes for this features with the setting name? That would make it more searchable (although search is not yet supported):
https://github.com/rails/rails/blob/e25d738430bdc6bdd04cd28be705484ea953e74e/guides/source/7_1_release_notes.md?plain=1#L869

Reply:
> If we show the version it was released, for every feature, the guides would become unreadable.

agreed. I have added a link instead, given it is not searchable.

> Maybe expand the 7.1 release notes for this features with the setting name? That would make it more searchable (although search is not yet supported):

I have done as suggested in principle, but altered the wording and force pushed:

`Add support for logging background job enqueue callers via `config.active_job.verbose_enqueue_logs`.`

If this is not conducive to the goals of the docs pls have no hesitation in alerting or correcting me.